src/dispatch/plugins/dispatch_google/docs/plugin.py

```diff
@@ -23,7 +24,80 @@ def remove_control_characters(s):
     return "".join(ch for ch in s if unicodedata.category(ch)[0] != "C")
 
 
-def replace_text(client: Any, document_id: str, replacements: List[str]):
+def find_links(obj: Any, find_key: str) -> Iterator[List[Any]]:
```

Comment:
```suggestion
def find_links(obj: dict | list, find_key: str) -> Iterator[list[Any]]:
```

Iterator should be imported from `_collections_abc` i believe.